src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -32,24 +31,20 @@
 	private static final Logger LOGGER = LoggerFactory.getLogger(JsonConvert.class);
 
 	/**
-	 * The method return the default path to JSON file.
-	 * 
-	 * @return <i>Path</i> where to read and write JSON.
+	 * The default path to JSON file.
 	 */
-	private static final Path apartmentPathJson() {
-		Path path = Paths.get("Apartment_Json.json");
-		return path;
-	}
+	public static final Path APARTMENT_PATH_JSON = Path.of("Apartment_Json.json");
 
 	/**
 	 * The method return a default JSON file read by jsonToApartments.
 	 * 
 	 * @return <i>Path</i> where jsonToApartments will read.
+	 * @throws URISyntaxException if the resource cannot be found
 	 */
-	private static final Path startApartment() {
-		Path path = Paths
-				.get("src/main/resources/io/github/oliviercailloux/y2018/jsonResources/defaultJsonToApartments.json");
-		return path;
+	private static final Path startApartment() throws URISyntaxException {
```

Comment:
Cette exception ne peut pas être lancée, en pratique, et ne devrait donc pas être annoncée. Vous pouvez ici capturer l’exception et la transformer en une VerificationException, par exemple (à ne pas annoncer). (C’est en gros ce que vous avez fait, mais à un meilleur endroit car échec plus rapide et en mieux factorisé.)

Reply:
Fix in commit 12d1c12213381bc05972b0cc51c70be71f83df47